CustomCraftSML/Interfaces/ICustomFood.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Collections.Generic;
+using Common.EasyMarkup;
+using CustomCraft2SML.Serialization.Components;
+
+namespace CustomCraft2SML.Interfaces
+{
+    interface ICustomFood : ITechTyped
+    {
+        short AmountCrafted { get; }
+        bool ForceUnlockAtStart { get; }
+
+        IList<EmIngredient> Ingredients { get; }
+        IList<string> LinkedItemIDs { get; }
+        IList<string> Unlocks { get; }
+        IList<string> UnlockedBy { get; }
+
+        string DisplayName { get; }
+        string Tooltip { get; }
+        TechType SpriteItemID { get; }
```

Comment:
Currently, the `Craftable` class requires an icon to be provided.  
We can provide a small set of sample icons. But we won't offer a `SpriteItemID` for this first release.